alternator/streams.cc

```diff
@@ -405,6 +396,9 @@ static std::chrono::seconds confidence_interval(const database& db) {
     return std::chrono::seconds(db.get_config().alternator_streams_time_window_s());
 }
 
+// Dynamo docs says no data shall live longer than 24h.
+static constexpr auto dynamo_streams_max_window = 24h;
```

Comment:
Oh, found it :-) The patches are in the wrong order (not bisectable).
As I mentioned in my earlier comment, please rename this constant (alternator, probably, or dynamodb if you must?) and please also think how we can use the same constant when configuring the CDC log TTL.